### ISSUE TITLE
MLS-Sola Winley (Short Post)

### ISSUE THREAD

**Issue by jaahmuhl** [promazo/Content-Team#62]
Raw Content: https://drive.google.com/drive/folders/1fd8I8It6LuOIMESkZmUlfCBJapuOXepZ?usp=sharing

**Comment by jaahmuhl** [comment]
Raw Content: https://drive.google.com/drive/folders/1fd8I8It6LuOIMESkZmUlfCBJapuOXepZ?usp=sharing

**Comment by jaahmuhl** [comment]
Transcript: https://www.rev.com/transcript-editor/shared/FIDQ-a0lT15bbO8UuKceGEFwy2dtNVWmfiyw3hr3nHqkHjgJJUH-UiIZSC-JLEN5Hjpn1pgKRILdnk4SOkIkNcuYejY?loadFrom=SharedLink

**Comment by James-Moore18** [comment]
Clip: 9:03-9:20, 9:48-10:13
IG Caption - Time is a finite resource increasingly taken up by responsibilities the older we get. Listen to MLS Chief Diversity and Inclusion Officer, Sola Winley, speak about the importance of using your time wisely and dedicating it to something you genuinely care about.

TikTok Caption: Listen to MLS Chief Diversity and Inclusion Officer, Sola Winley, speak about the importance of using your time wisely and dedicating it to something you genuinely care about.

On-screen Caption: Spend your time WISELY! You won't get it back!


Interviewer: Brett Hummel
Videographer: Bryan Acevedo
#promazo #reallifemba #promazoexecutiveinterviews #goals #dream #diversity #leadership #mls

**Comment by bryanace53** [comment]
Only have one angel

**Comment by jaahmuhl** [comment]
@bryanace53 Yep this one we'll need to do like an audio thing where we show the sounds going up and down or something? Or maybe like his picture zooming in and out of it or of MLS??

**Comment by bryanace53** [comment]
https://drive.google.com/drive/u/4/search?q=sola

**Comment by jaahmuhl** [comment]
![2022-07-27_18-02-04.png](https://images.zenhubusercontent.com/5cb3b35dd74bd07972cea86e/0e424efa-c8ee-4156-bf4d-453cfe226d4e)

**Comment by bryanace53** [comment]
https://drive.google.com/drive/u/4/folders/1nZGDEgS4yFKjlE7YZlFRQ5PdoUVaeFEh

Updated version

**Comment by jaahmuhl** [comment]
@bryanace53 
![2022-08-09_09-38-49.png](https://images.zenhubusercontent.com/5cb3b35dd74bd07972cea86e/5012d101-fc1b-493b-97f2-aa2abd34c67f)

**Comment by bryanace53** [comment]
Here is the new look let me know what you think! 

https://drive.google.com/drive/u/4/folders/1nZGDEgS4yFKjlE7YZlFRQ5PdoUVaeFEh

**Comment by jaahmuhl** [comment]
@bryanace53 

Can't hear first words
![2022-08-23_21-41-49.png](https://images.zenhubusercontent.com/5cb3b35dd74bd07972cea86e/854a9755-9d7d-4fec-8bc4-0a934d6940c3)

**Comment by jaahmuhl** [comment]
@bryanace53 

put his name and title back in when it is showing his face please

![2022-08-23_21-42-52.png](https://images.zenhubusercontent.com/5cb3b35dd74bd07972cea86e/0d6201f1-8129-4b92-9eec-645c064a6372)

**Comment by bryanace53** [comment]
https://drive.google.com/drive/u/4/folders/1nZGDEgS4yFKjlE7YZlFRQ5PdoUVaeFEh

He says um before he speaks so technically I can't push it back a little.
